app/api/testing/firebase-test-utils.ts

```
@@ -0,0 +1,147 @@
+  public async joinRoom(roomId: string): Promise<any> {
+    return {
+      userId: 'mock-user-id',
+      joined: Date.now(),
+    };
+  }
+
+  public async leaveRoom(roomId: string, userId: string): Promise<void> {}
+
+  public async sendSignal(roomId: string, message: any): Promise<void> {}
+
+  public async getSignals(roomId: string, since: number = 0): Promise<any[]> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)